packs/feats/spirit-guide-form.json

```diff
@@ -4,14 +4,14 @@
     "name": "Spirit Guide Form",
     "system": {
         "actionType": {
-            "value": "passive"
+            "value": "action"
         },
         "actions": {
-            "value": null
+            "value": 2
         },
         "category": "class",
         "description": {
-            "value": "<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p>You fuse with your spirit guide, becoming a hybrid being with enhanced physical and spiritual power. This effect lasts 1 minute; you can end it early as a free action. During this time, you can't use any other actions granted by this archetype. You gain the following statistics and abilities, regardless of your type of spirit guide.</p>\n<ul>\n<li>You become incorporeal.</li>\n<li>You gain resistance 10 to all damage (except force, ghost touch, or positive; double resistance vs. non-magical).</li>\n<li>Darkvision.</li>\n<li>An unarmed melee attack of the same type as your spirit guide (either claws or jaws), which is the only attack you can Strike with. You're trained with this attack. Your attack modifier is +29 or your normal unarmed attack modifier, whichever is higher. You use the listed statistics for the unarmed attack type:</li>\n<li><strong>Claws</strong> (agile, backstabber, finesse) ?, Damage 3d6+5 slashing plus 1d6 force;</li>\n<li><strong>Jaws</strong> (forceful) ? Damage 3d10+5 piercing plus 1d6 force.</li>\n</ul>"
+            "value": "<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p>You fuse with your spirit guide, becoming a hybrid being with enhanced physical and spiritual power. This effect lasts 1 minute; you can end it early as a free action. During this time, you can't use any other actions granted by this archetype. You gain the following statistics and abilities, regardless of your type of spirit guide.</p>\n<ul>\n<li>You become incorporeal.</li>\n<li>You gain resistance 10 to all damage (except force, ghost touch, or vitality; double resistance vs. non-magical).</li>\n<li>Darkvision.</li>\n<li>An unarmed melee attack of the same type as your spirit guide (either claws or jaws), which is the only attack you can Strike with. You're trained with this attack. Your attack modifier is +29 or your normal unarmed attack modifier, whichever is higher. You use the listed statistics for the unarmed attack type:\n<ul>\n<li><strong>Claws</strong> (agile, backstabber, finesse) <strong>Damage</strong> 3d6+5 slashing plus 1d6 force</li>\n<li><strong>Jaws</strong> (forceful) <strong>Damage</strong> 3d10+5 piercing plus 1d6 force.</li>\n</ul>\n</li>\n</ul>"
```

Comment:
The form should give only one strike based on a choice.

Reply:
True, but there isn't a way to currently handle this.

Reply:
Could we maybe get fancy with SpecialStatistic? We haven't mucked with that much yet.

Reply:
I'm not sure how to do this. Documentation and examples for SpecialStatistic are pretty sparse.

Reply:
yeah, it's on my list to remedy before i do the RE videos

Reply:
Now uses the new predicate support for BattleForm Strikes.